cvxpy/interface/scipy_wrapper.py

```diff
@@ -14,10 +14,18 @@
 limitations under the License.
 """
 
-from scipy.sparse import spmatrix
+from scipy import sparse
 
 from cvxpy.expressions import expression as exp
 
+SPARSE_MATRIX_CLASSES = [
+    sparse.csc_matrix, 
+    sparse.csr_matrix, 
+    sparse.coo_matrix, 
+    sparse.bsr_matrix, 
+    sparse.lil_matrix, 
+    sparse.dia_matrix
```

Comment:
```suggestion
    sparse.dia_matrix,
    sparse.dok_matrix,
```

I forgot `dok` in my comment